### DISCUSSION TITLE
How to match a word at the end of a line with `consult-line`?

### DISCUSSION THREAD

**Discussion by richardfrangie** [minad/consult#1139] [Q&A] [answered]
Hi, is there a way to match a word at the end of a line using `consult-line`? I am using `consult` and `orderless` with the same settings that come out in the repos `(setq completion-styles '(orderless basic))`.

For example:

```
first second third
first third second
```

I tried with `third$`, but it doesn't match any. If I try with `third` it matches both.

Thanks.

**Comment by minad** [comment, marked as answer]
You can use the regexp `third$` but for technical reasons additional Orderless configuration is needed. See these examples:

* https://github.com/minad/consult/wiki#orderless-style-dispatchers-ensure-that-the--regexp-works-with-consult-buffer
* https://github.com/minad/consult/wiki#minads-orderless-configuration
